synfig-studio/src/synfigapp/actions/waypointsetsmart.h

```diff
@@ -59,9 +59,12 @@ class WaypointSetSmart :
 
 	void calc_waypoint();
 	void enclose_waypoint(const synfig::Waypoint& waypoint);
+	void redirect_to_value_desc_set_action();
 
 	std::set<synfig::Time> times;
 
+	ValueDesc value_desc;
+
```

Comment:
keep it next to `value_node` declaration.

Reply:
done